package.json

```diff
@@ -30,6 +30,7 @@
 		"eslint-plugin-svelte3": "^4.0.0",
 		"ethers": "^6.6.2",
 		"firebase": "^9.23.0",
+		"hardhat": "^2.16.1",
```

Comment:
I tried npx hardhat and it seems that it only works if it is installed. We can use something else, this just worked for me at first without being familiar with it. 

```bash
% npx hardhat node
888    888                      888 888               888
888    888                      888 888               888
888    888                      888 888               888
8888888888  8888b.  888d888 .d88888 88888b.   8888b.  888888
888    888     "88b 888P"  d88" 888 888 "88b     "88b 888
888    888 .d888888 888    888  888 888  888 .d888888 888
888    888 888  888 888    Y88b 888 888  888 888  888 Y88b.
888    888 "Y888888 888     "Y88888 888  888 "Y888888  "Y888

Welcome to Hardhat v2.16.1

✔ What do you want to do? · Create an empty hardhat.config.js
Config file created

You need to install hardhat locally to use it. Please run:

pnpm add -D "hardhat@^2.16.1"


Give Hardhat a star on Github if you're enjoying it!

     https://github.com/NomicFoundation/hardhat
Start time 21:34:55, end time 21:35:15, elapsed 20s

% npx hardhat node
Error HH12: Trying to use a non-local installation of Hardhat, which is not supported.
Please install Hardhat locally using npm or Yarn, and try again.

For more info go to https://hardhat.org/HH12 or run Hardhat with --show-stack-traces
```

Reply:
Alternatively, you can fix it by adding script to the `package.json` which knip will detect. Something like `"start:blockchain": "hardhat"`

Reply:
> Alternatively, you can fix it by adding script to the `package.json` which knip will detect. Something like `"start:blockchain": "hardhat"`

That did not work for me, I still had to ignore hardhat to avoid errors.